.github/workflows/test.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       - name: Install dependencies
         run: |
           ./.github/scripts/install_deps.sh
-          brew install stylua uncrustify
+          brew install stylua
```

Comment:
brew can be really slow in CI, so it might even worth fetching their binary instead of using `brew` in CI.

Reply:
Good call. The lint times are getting gross so will need to make a more thorough refactor of this, one of which is to dump brew 👍

Reply:
Follow-up PR though

Reply:
If you do
```sh
export HOMEBREW_NO_AUTO_UPDATE=1
export HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK=1
export HOMEBREW_NO_INSTALL_UPGRADE=1
export HOMEBREW_NO_INSTALL_CLEANUP=1
brew untap homebrew/core
```
this should substantially speed up your usage of `brew`.

Reply:
Didn't make a difference.